distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -577,6 +585,25 @@ public void testCswXmlIngest() {
     }
 
     @Test
+    public void testCswXmlWithHeaderIngest() {
+        Response response = ingestXmlWithHeaderMetacard();
+        ValidatableResponse validatableResponse = response.then();
+
+        validatableResponse.body(hasXPath("//TransactionResponse/TransactionSummary/totalInserted",
```

Comment:
I could switch it to `response.then().body(hasXPath...`, but I'm not sure that I can remove the `then()`.